js/app.js

```diff
@@ -274,7 +274,7 @@ const generateBingoCards = async () => {
   for (let i = 0; i < 25; ) {
     let n = randomNumber(0, phrases.length-1);
     if (usedNumbers.includes(n)) {
-      n = randomNumber(0, 24);
+      n = randomNumber(0, phrases.length-1);
```

Comment:
Could we assign the phrases.length-1 to an variable to avoid .length being called each time?

Reply:
Sounds like a good idea! Will add that later.

Reply:
Added a variable for that